guides/tutorials/agent-config.md

```diff
@@ -200,6 +200,18 @@ Stringified JSON object of the transaction.
 
 JSON representation specifying the version and acceptance mechanism. The version is the unique version of the transaction author agreement acceptance mechanism list (AML). The acceptance mechanism refers to the acceptance mechanism label of the item in the AML. For more details you may consult the [indy-node docs on AML](https://github.com/hyperledger/indy-node/blob/master/docs/source/transactions.md#transaction_author_agreement_aml)
 
+### `indyLedgers.transactionAuthorAgreement.version`
```

Comment:
Should this have four `####`? To indicate the extra level of nesting

Reply:
Yeah I think that that would be best.

Reply:
@morrieinmaas back to you :)